TSPL.docc/ReferenceManual/Statements.md

```diff
@@ -958,11 +958,11 @@ evaluates to `true` at compile time.
 The *compilation condition* can include the `true` and `false` Boolean literals,
 an identifier used with the `-D` command line flag, or any of the platform
 conditions listed in the table below.
-
+**`OSX`** | **`macOS`** | **`tvOS`** | **`watchOS`** | **`iOS`** | **`Linux`** | **`FreeBSD`** | **`OpenBSD`** | **`Windows`** | **`Android`** | **`PS4`** | **`Cygwin`** | **`Haiku`** | **`WASI`** \
```

Comment:
```suggestion

```

Looks like you might have duplicated this line accidentally.

Reply:
Yes. Thanks for pointing out. Fixed